### ISSUE TITLE
Does Ghostery Lite for Safari block Youtube ads?

### ISSUE THREAD

**Issue by RyanNgCT** [ghostery/GhosterySafari#9]
Question above. Ghostery Lite seems to not block Youtube ads now (but it used to), any reason for this?

**Comment by RyanNgCT** [comment]
@fcjr @christophertino I got contacted by support so I suppose this is a known issue across other ad-blockers too?

![image](https://user-images.githubusercontent.com/48358569/118906866-26d64c00-b951-11eb-862e-fb719075b590.png)

**Comment by RyanNgCT** [comment]
Hi I would just like to follow up on this issue. Thanks!

**Comment by christophertino** [comment]
Sorry for the delay. We should still be blocking youtube ads. This project has been parked for a bit, but I'll be releasing a new version with updated ad filters to hopefully stay current with youtube's latest patterns.

**Comment by RyanNgCT** [comment]
Do update me when this version has been released. Thanks!

**Comment by christophertino** [comment]
The release is pending App Store review. In the meantime, try cycling the Ghostery network and cosmetic content blockers in Safari to see if that helps. From Safari > Preferences > Extensions > uncheck and recheck Network Filters and Cosmetic Filters for Ghostery Lite.  I was catching YouTube ads after doing that.

**Comment by RyanNgCT** [comment]
@christophertino The ads stopped for you? Downloaded the latest version as you commented and still seeing ads...

![image](https://user-images.githubusercontent.com/48358569/129730048-930e0cf8-42d8-4f36-b3ee-ce79a66a0a51.png)

I'm using Safari 14.1.1.

![image](https://user-images.githubusercontent.com/48358569/129730124-252afd1b-a48c-48d1-be86-d5308b0fb74b.png)

**Comment by christophertino** [comment]
Have you tried cycling the content blocker extensions like I mentioned above? That seemed to do the trick for me.

**Comment by RyanNgCT** [comment]
@christophertino yes.. its on and off... quite annoying to have to cycle through the settings every few minutes. Is it possible to  add code that will work somewhat like uBlock Origin? Thanks!
